plugins/postfix/postfix_stats

```diff
@@ -59,7 +64,9 @@ GPLv2
 =cut
 
 #set -xv
-SYS_LOG="${logfiles:-/var/log/syslog /var/log/syslog.0}"
+MAIL_LOG="${logfiles:-/var/log/mail.log /var/log/mail.log.1}"
+
+GREP_HOSTNAME=$(echo $0 | sed -r 's/^.+postfix_stats_?//')
```

Comment:
Please take care for proper quoting (here `"$0"`). See the output of `shellcheck -s dash postfix_stats` for details.
Maybe `GREP_HOSTNAME=$(basename "$0" | sed -r '^postfix_stats_?//')` would be even more precise?